Enrique/principal vs secondary

Regis18 opened the pull request:
Principal vs secondary: There are two files of Java

I create two files of Java in order to resolve this problem, the Diagonal file is with the methods and the file Main is the executable of this program

Regis18 committed:
cambios

Regis18 committed:
practic part 2

J0c4 commented on src/principal-vs-secondary/regis/Diagonal.java:
```diff
@@ -0,0 +1,66 @@
+import java.util.Random;
+public class Diagonal {
```
Let's add class documentation: brief description of class responsability/purpose

J0c4 commented on src/principal-vs-secondary/regis/Diagonal.java:
```diff
@@ -0,0 +1,66 @@
+import java.util.Random;
```
Let's keep a space between class definition and imports block

J0c4 commented on src/principal-vs-secondary/regis/Diagonal.java:
```diff
@@ -0,0 +1,66 @@
+import java.util.Random;
+public class Diagonal {
+
+    public int [][] matrixRandom(int side){
```
Let's add docs for methods

J0c4 commented on src/principal-vs-secondary/regis/Diagonal.java:
```diff
@@ -0,0 +1,66 @@
+        return matrix;
+    }
+
+    public int addDiagonalMain(int [][] matrix){
```
This method name could be a little bit confusing. If you use it from an instance of this class, it will look like `diagonal.addDiagonalMain(matrix)`, which seems to say `Add a diagonal matrix to the diagonal instance`

J0c4 commented on src/principal-vs-secondary/regis/Diagonal.java:
```diff
@@ -0,0 +1,66 @@
+        return result;
+    }
+
+    public int addDiagonalSecondary(int [][] matrix){
```
Could this method and the previous one be one single method? maybe its difference could be handled by a single `if-then-else` and a parameter flag

J0c4 commented on src/principal-vs-secondary/regis/Diagonal.java:
```diff
@@ -0,0 +1,66 @@
+        if (resultMain  >  resultSecondary) {
+            message = "Principal Diagonal win!";
+        }
+        else {
```
Let's use 
```
} else {
```
instead of
```
} 
else {
```

J0c4 commented on src/principal-vs-secondary/regis/Diagonal.java:
```diff
@@ -0,0 +1,66 @@
+        else {
+            message = "Secondary Diagonal win!";
+        }
+        if(resultMain  == resultSecondary){
```
space between `if` and `(` missing. Same between `)` and `{`

J0c4 commented on src/principal-vs-secondary/regis/Diagonal.java:
```diff
@@ -0,0 +1,66 @@
+            message = "Draw!";
+        }
+        System.out.println(message);
+        //return message;
```
If this commented line is not needed, then let's remove it

Regis25 committed:
Modifications of the principal-vs-secondary

I did modifications of the class  of principal-vs-secondary
about some commits of methods and resolutions on the code
